src/app/garfile/departure/index.njk

```diff
@@ -186,6 +207,7 @@
     selectElement: document.querySelector('#departurePort'),
     minLength: 2,
   });
+  document.getElementById("departurePort").setAttribute('aria-describedby', 'departurePort-hint{{ ' departurePort-error' if errors | containsError('departurePort') }}');
```

Comment:
Javascript here to add the aria-describedby attribute to the accessible autocomplete. This is something that probably the component itself should have done anyway...